assets/popup.css

```diff
@@ -72,19 +72,34 @@
   }
 }
 
-html[data-theme] {
-  background-color: var(--color-background-default, white) !important;
-  color: var(--color-font-body, black);
+@keyframes spin {
+  0% {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate(360deg);
+  }
 }
 
-html:not([data-theme]) {
-  background-color: var(--backgroundColor, var(--defaultBackgroundColor, white));
-  color: var(--textColor, var(--defaultTextColor, black));
+html {
+  background: var(--defaultBackgroundColor);
+  color: var(--defaultTextColor);
+}
+
+html[data-theme="light"] {
+  --defaultBackgroundColor: white;
+  --defaultTextColor: black;
+}
+
+html[data-theme="dark"] {
+  --defaultBackgroundColor: black;
+  --defaultTextColor: white;
```

Comment:
This change is great, but it's breaking the theme in BE because `data-theme` is not set for all BE screens.
We can modify `~/assets/popup.js` so this .css works, retrieving the theme from the extension's local storage with something like this:

```js
async function setTheme() {
  const result = await chrome.storage.local.get(['setting_display_theme']);
  let theme = result.setting_display_theme || 'system';
  if (typeof theme === 'string' && theme.startsWith('"')) 
    theme = JSON.parse(theme);
  document.documentElement.dataset.theme = theme;
}

setTheme();
```

The only downside with this approach is that BE doesn't receive the theme as a param, so the async nature of retrieving from extension storage makes this slower than the iframe param-based approach.

Reply:
Fixed, but no need to change the JS. I messed up the selectors so BE was not getting the right CSS variables applied.